nmdc_server/utils.py

```diff
@@ -34,3 +36,48 @@ def log_extras(req: Request) -> Dict[str, Any]:
         "method": req.method,
         "url": req.url,
     }
+
+
+def sanitize_filename(filename: str) -> str:
```

Comment:
should this be moved into the `storage.py` module? at first glance it seems that this is only used to adhere to GSC requirements.

Reply:
also too bad that the `google-cloud-storage` library doesn't seem to provide this functionality

Reply:
> should this be moved into the storage.py module? 

Yes, that's a good idea. Will do.

> too bad that the google-cloud-storage library doesn't seem to provide this functionality

Couldn't agree more.